external/opensocdebug/hardware/modules/system_diagnosis/common/source/diagnosis_config.vh

```diff
@@ -0,0 +1,114 @@
+/* Copyright (c) 2012-2013 by the author(s)
```

Comment:
@wallento worked hard to get all defines removed from optimsoc. What's the configuration story for OSD? Which of the defines are actually needed, which ones can be passed as parameters through the system, or do we need to adopt a config_t struct similar to optimsoc?